java/dataset/src/test/java/org/apache/arrow/dataset/CsvJsonWriteSupport.java

```diff
@@ -24,17 +24,18 @@
 import java.net.URISyntaxException;
 import java.util.Random;
 
-public class CsvWriteSupport {
+public class CsvJsonWriteSupport {
```

Comment:
How about using `TextBasedWriteSupport` or something?
If we use this style, we may need to add more formats to class name such as `CsvJsonXXXYYYWriteSupport` in the future.